lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java

```diff
@@ -224,14 +224,15 @@ else if ("shortanswer".equals(questionType) &&
 		}
 		
 		if (notSubmitted.size() > 0) {
-		    List<String> missing = new ArrayList<String>();
-		    for (String userId: notSubmitted) {
-			try {
-			    missing.add(UserDirectoryService.getUser(userId).getDisplayName());
-			} catch (Exception e) {
-			    missing.add(userId);
+			List<String> missing = new ArrayList<String>();
```

Comment:
same spacing issue

Reply:
Same as above, the spacing is ok after the changes.